src/shared/bus-locator.c

```diff
@@ -69,6 +69,12 @@ const BusLocator* const bus_hostname = &(BusLocator){
         .interface = "org.freedesktop.hostname1"
 };
 
+const BusLocator* const bus_systemd_loc = &(BusLocator){
```

Comment:
This is a duplicate of `bus_systemd_mgr`

Reply:
Thank you for being very patient sir.